app/main.ipynb

```diff
@@ -56,31 +57,51 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
+   "execution_count": 2,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "8056"
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "a = [0] * 1000\n",
+    "sys.getsizeof(a)"
    ],
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T12:02:41.673421Z",
-     "start_time": "2023-07-03T12:02:41.666277Z"
+     "end_time": "2023-10-11T11:14:50.683458900Z",
+     "start_time": "2023-10-11T11:14:50.624648500Z"
     }
    }
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "outputs": [],
+   "execution_count": 3,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "1000"
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your code here (for numpy syntax)"
+    "arr = np.zeros(1000, dtype=np.int8)\n",
+    "arr.nbytes"
```

Comment:
You should use sys library for printing the size of the array as stated in the task